src/dotnet/APIView/ClientSPA/src/app/_workers/apitree-builder.worker.ts

```diff
@@ -67,7 +68,8 @@ function buildCodePanelRows(nodeIdHashed: string, navigationTree: NavigationTree
   let buildChildren = true;
   let addNodeToBuffer = false
  
-  if (nodeIdHashed !== "root" && (apiTreeBuilderData?.diffStyle === "trees" || apiTreeBuilderData?.diffStyle === "nodes") && !node.isNodeWithDiffInDescendants) {
+  if (nodeIdHashed !== "root" && (apiTreeBuilderData?.diffStyle === "trees" || apiTreeBuilderData?.diffStyle === "nodes") && 
+    (!node.isNodeWithDiffInDescendants || (!apiTreeBuilderData?.showDocumentation && node.isNodeWithNoneDocDiffInDescendants == false))) {
```

Comment:
```suggestion
    (!node.isNodeWithDiffInDescendants || (!apiTreeBuilderData?.showDocumentation && !node.isNodeWithNoneDocDiffInDescendants))) {
```

super nit: is using `!` clearer than the `==` check here?